diff --git a/node_modules/@shy1118/mammoth/lib/docx/body-reader.js b/node_modules/@shy1118/mammoth/lib/docx/body-reader.js
index a792c80..816f769 100644
--- a/node_modules/@shy1118/mammoth/lib/docx/body-reader.js
+++ b/node_modules/@shy1118/mammoth/lib/docx/body-reader.js
@@ -441,10 +441,12 @@ function BodyReader(options) {
         return readXmlElements(element.children).map(function (children) {
             var properties = element.firstOrEmpty("w:tcPr");
 
+            var bgColor = readColor(properties.firstOrEmpty("w:shd"))
+
             var gridSpan = properties.firstOrEmpty("w:gridSpan").attributes["w:val"];
             var colSpan = gridSpan ? parseInt(gridSpan, 10) : 1;
 
-            var cell = documents.TableCell(children, { colSpan: colSpan });
+            var cell = documents.TableCell(children, { colSpan: colSpan, bgColor: bgColor });
             cell._vMerge = readVMerge(properties);
             return cell;
         });
